apps/docs/src/components/document/common/mdx.tsx

```diff
@@ -64,6 +64,7 @@ import { ColorPrimitives } from '@/components/document/color/color-primitives';
 import { TwoThirds, TwoThirdsOneThird } from '@/components/layouts/two-thirds';
 import { Highlight } from '@/components/typography/highlight';
 import { Link } from '@/components/typography/link';
+import { List } from '@/components/typography/list';
 
```

Comment:
I think you can directly import the `List` component from `@govie-ds/react` without having to create a new component in the docs repo

Reply:
I'm not sure about the others. If they have some other functionality added to them I think they can stay like this